galaxy_ng/app/api/ui/serializers/__init__.py

```diff
@@ -64,5 +66,7 @@
     'ContainerManifestSerializer',
     'ContainerManifestDetailSerializer',
     'ContainerReadmeSerializer',
-    'ContainerNamespaceDetailSerializer'
+    'ContainerNamespaceDetailSerializer',
+    'ContainerRegistryRemote',
```

Comment:
we shouldn't need the model passed through the serializers, and it doesn't look like it is used in this PR, so it and the import can be removed.